README.md

```diff
@@ -1,5 +1,8 @@
 ## Neugram
 
+[![Build Status](https://secure.travis-ci.org/neugram/ng.png)](http://travis-ci.org/neugram/ng)
+[![GoDoc](https://godoc.org/neugram.io/ng?status.svg)](https://godoc.org/neugram.io/ng)
```

Comment:
There isn't anything to see on godoc.org, and I'm not sure I want to put too much documentation there. How about skipping this badge?

Reply:
done.